RepositoryCompiler/CodeModel/CaDETModel/CodeItems/CaDETClass.cs

```diff
@@ -28,6 +28,7 @@ public string ContainerName
         public List<CaDETMember> Members { get; internal set; }
         public List<CaDETField> Fields { get; internal set; }
         public CaDETClassMetrics Metrics { get; internal set; }
+        public ISet<CaDETMember> InvokedMethods { get; internal set; }
```

Comment:
I wouldn't calculate this during parsing, nor store it in a field. There is no reason to maintain a set of class level invoked methods apart from the metric calculation - which is why we should move the code that is currently in the CodeParser to the MetricCalculator, get the set, count the elements, and let it be garbage collected.